js/demo.js

```diff
@@ -0,0 +1,211 @@
+        leftPizzaOnThePage.clicks++;
+    }
+    //counting the clicked on image and we only one of this ids per click
+    if (event.target.id === 'right_pizza_img') {
+        rightPizzaOnThePage.clicks++;
+    }
+    //handle viewing the previous images
+    const tempPickedPizzas = [];
+
+    let leftPizzaIndex;
+    do {
+        leftPizzaIndex = Math.floor(Math.random() * allPizzas.length);
+    } while (previouslyPickedPizzas.includes(allPizzas[leftPizzaIndex]) ||
+        tempPickedPizzas.includes(allPizzas[leftPizzaIndex]));
+    tempPickedPizzas.push(allPizzas[leftPizzaIndex]);
+
+    let rightPizzaIndex;
+    do {
+        rightPizzaIndex = Math.floor(Math.random() * allPizzas.length);
+    } while (previouslyPickedPizzas.includes(allPizzas[rightPizzaIndex]) ||
+        tempPickedPizzas.includes(allPizzas[rightPizzaIndex]));
+    tempPickedPizzas.push(allPizzas[rightPizzaIndex]);
+
+    leftPizzaOnThePage = allPizzas[leftPizzaIndex];
+    rightPizzaOnThePage = allPizzas[rightPizzaIndex];
+    console.log(leftPizzaOnThePage, rightPizzaOnThePage);
+    //now update on the page
+    //clickable new images being shown.
+    console.log(leftPizzaOnThePage.imageSrc, rightPizzaOnThePage.imageSrc);
+    leftPizzaImage.src = leftPizzaOnThePage.imageSrc;
+    rightPizzaImage.src = rightPizzaOnThePage.imageSrc;
+
+    //handle the previously picked pizzas
+    previouslyPickedPizzas = [];
+    previouslyPickedPizzas.push(allPizzas[leftPizzaIndex]);
+    previouslyPickedPizzas.push(allPizzas[rightPizzaIndex]);
+    console.log('previous', previouslyPickedPizzas);
+
+    console.log('Tot clicks', totalClicks);
+    if (totalClicks === 25) {
+        pizzaImageSectionTag.removeEventListener('click', handleClickOnPizza);
+    }
+}//closes our function
+
+
+//create a function handleResult list showing ul and li on the left side of the page
+function handleResultsList() {
+    // console.log('proof from handle results list lis');
+    document.getElementById('pizza-clicks-list').style.background = '#8197c9';
+    document.getElementById('pizza-clicks-list').style.color = 'whitesmoke';
+    let ul = document.getElementById('pizza-clicks-list');
+    for (let i = 0; i < allPizzas.length; i++) {
+        let currentPizza = allPizzas[i];
+        let li = document.createElement('li');
+        li.textContent = currentPizza.pizzaName + ' got ' + currentPizza.clicks + ' votes';
+        // console.log('li: ',li);
+        ul.appendChild(li);
+    }
+}
+
+
+
+//Eventually add the chart here form canvas.js big concept not much to code.
+function handleChartResults() {
+    // console.log('proof from handle chart results');
+    //going to call the createChart function/
+    makeAPizzaChart();
+}
+
+//add our event listeners for our button clicks  = 'click' events functions
+//html element. method (event looking for,  name of our() that handles the event)
+pizzaImageSectionTag.addEventListener('click', handleClickOnPizza);
```

Comment:
Modify your event listener attachment to target the images by class

Ex.
document.querySelectorAll('.product-img').forEach(function (img) {
    img.addEventListener('click', handleClickOnProduct);
});